src/collector.ts

```diff
@@ -31,9 +35,10 @@ interface IDependencyCollector {
 
 /* Dependency class that can be created from `IKeyValueEntry` */
 class Dependency implements IDependency {
-  name:    IPositionedString;
-  version: IPositionedString;
-  constructor(dependency: IKeyValueEntry) {
+  name:             IPositionedString;
+  version:          IPositionedString;
+  version_prefix:   string
+  constructor(dependency: IKeyValueEntry, version_prefix: string) {
```

Comment:
You can keep "" as default value to version_prefix, so that you can avoid modifying exiting code.